rtb/verbali/esterni/VE_2024-12-13_ver1.0.0.typ

```diff
@@ -0,0 +1,75 @@
+    "Davide Picello", 
+    "Klaudio Merja",
+    [- Stesura del verbale],
+  ),
+  content: content,
+  firmaRichiesta: true,
+)
+
+
+#infoRiunione(luogo: "Google Meet", data: "13/12/2024", ora: "10:30", durata: "30 minuti", partecipantiEsterni: ("Andrea Dorigo",))
+
+
+
+= Sintesi dell'incontro 
+L'incontro ha ufficialmente segnato la fine del secondo _#rifGlossario("sprint")_, ed ha visto il gruppo presentare per la prima volta una proposta concreta del simulatore dati con interfaccia, utilizzando le tecnologie concordate.
+
+In particolare è stata mostrata la progettazione del _#rifGlossario("database")_, presentando il modello entità-relazione, a cui è seguito un breve confronto tra il gruppo ed il partecipante esterno, con esito positivo. 
+
+//Mostrato Nifi + Grafana
+Infine è stato esposto il generatore dati ed il suo funzionamento mostrando i flussi di dati in #rifGlossario("Apache NiFi"), e successivamente l'interfaccia realizzata con #rifGlossario("Grafana"). Quest'ultima presentava una mappa in cui venivano indicati i punti geospaziali creati dal generatore. Queste coordinate simulavano un percorso di un utente in bicicletta in modo corretto e coerente, come possiamo notare nelle seguenti immagini.
+
+#figure(image("../../../assets/img/verbali/simGrafana1.png", height: 30%), caption: [Esempio di visualizzazione su Grafana dei dati generati dal simulatore])
+
+#figure(image("../../../assets/img/verbali/simGrafana2.png", height: 30%), caption: [Esempio di visualizzazione su Grafana, in modalità _route_, dei dati generati dal simulatore])
+
+
+= Risposte alle domande  
+Durante l'incontro sono state poste alcune domande, che vengono riportate qui di seguito.
+
+== Migrazione da Apache NiFi
+Durante l'implementazione del simulatore dati sono stati riscontrate alcune problematiche durante nell'utilizzo di Apache NiFi. Queste derivano dalla curva d'apprendimento di questo applicativo rispetto all'utilizzo effettivamente richiesto per questo progetto. Il gruppo si è confrontato con l'azienda a riguardo, ed è emersa la possibilità di migrare verso altri servizi come Flink o #rifGlossario("Spring"). Il gruppo, entro il prossimo incontro, prenderà in considerazione se continuare ad utilizzare il _software_ utilizzato attualmente o se, invece, sarà più vantaggioso sostituirlo.
+
+
+//2. Come fare a implementare logiche più complesse con nifi? -> come implementare pattern per Cardin
+== Implementazione di logiche complesse con Apache NiFi
+Un'altra domanda posta dal gruppo è stata se Apache NiFi si presti all'implementazione di logiche complesse e di design pattern, in vista del colloquio sulle tecnologie che il gruppo dovrà sostenere per il superamento dell'esame. L'azienda ci ha rassicurato, dicendoci che in genere è sufficiente riconoscere queste logiche nei _software_ che utilizziamo. Per chiarire ogni dubbio il gruppo porrà questa domanda direttamente al professor Cardin durante l'incontro fissato per giovedì 19 dicembre.
```

Comment:
...design pattern, in vista del colloquio... toglierei la virgola.

Non ho capito invece a qualche colloquio sulle tecnologie fai riferimento e quali siano queste logiche nel software che utilizziamo onestamente. Se riesci a spiegarmelo meglio

Reply:
> Non ho capito invece a qualche colloquio sulle tecnologie fai riferimento e quali siano queste logiche nel software che utilizziamo onestamente. Se riesci a spiegarmelo meglio

Intendevo il colloquio dell'RTB (ora mi viene il dubbio che il colloquio fosse solo al PB)

Reply:
Eviterei. Non ho mai sentito riferimenti espliciti ad un colloquio o altro. Magari me lo son perso io

Reply:
Ah. Ho visto la correzione: più che valutazione didattica (mi pare inopportuno metterlo), metterei per la creazione del MVP